interactors/nonceHandlerV3/addressNonceHandler.go

```diff
@@ -54,19 +57,20 @@ func NewAddressNonceHandlerV3(proxy interactors.Proxy, address sdkCore.AddressHa
 
 // ApplyNonceAndGasPrice will apply the computed nonce to the given FrontendTransaction
 func (anh *addressNonceHandler) ApplyNonceAndGasPrice(ctx context.Context, txs ...*transaction.FrontendTransaction) error {
-	nonce, err := anh.fetchNonce(ctx)
-	for i, tx := range txs {
-		tx.Nonce = nonce + uint64(i)
+	for _, tx := range txs {
+		//anh.mut.Lock()
```

Comment:
why the commented code with lock/unlock?

Reply:
forgot about it. thanks :+1: